syntax = "proto3";

package grpc.health.v1;

import "google/protobuf/empty.proto";

service Health {
  rpc Check (google.protobuf.Empty) returns (HealthCheckResponse);
}

message HealthCheckResponse {
  enum ServingStatus {
    UNKNOWN = 0;
    SERVING = 1;
    NOT_SERVING = 2;
  }
  ServingStatus status = 1;
}
